kroxylicious-api/src/main/java/io/kroxylicious/proxy/filter/FilterFactory.java

```diff
@@ -23,7 +23,7 @@ public interface FilterFactory<C, I> {
 
     /**
      * Initializes the factory with the specified configuration.
-     * This method is guaranteed to be called at most once, and before any call to
+     * This method is guaranteed to be called at most once, per factory instance, and before any call to
```

Comment:
Thinking about this further based on the tests I ended up writing I think this is still not quite right.